### PR TITLE
Rename the package from whisk to org.apache.openwhisk

### PR DESCRIPTION
**Pull request by houshengbo**
## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

File: common/scala/src/main/scala/org/apache/openwhisk/core/WhiskConfig.scala [hunk 1]
```diff
@@ -186,56 +186,56 @@ object WhiskConfig {
 }
 
 object ConfigKeys {
-  val cluster = "whisk.cluster"
-  val loadbalancer = "whisk.loadbalancer"
-  val buildInformation = "whisk.info"
+  val cluster = "org.apache.openwhisk.cluster"
```
Comment (chetanmeh) [inline common/scala/src/main/scala/org/apache/openwhisk/core/WhiskConfig.scala]: I do not think we need to change the configKeys?

Comment (houshengbo) [reply common/scala/src/main/scala/org/apache/openwhisk/core/WhiskConfig.scala]: We need to change everything based on the Apache policy.

Comment (dgrove-oss) [reply common/scala/src/main/scala/org/apache/openwhisk/core/WhiskConfig.scala]: I'm not convinced we need the extra levels here.  It should be further discussed.

Even if we eventually decide to make the change, it absolutely should be a separate PR.  Changing the name of config variables is significantly higher impact on downstream projects than changing the names of the Scala/Java packages

File: common/scala/src/main/scala/org/apache/openwhisk/core/WhiskConfig.scala [hunk 2]
```diff
@@ -30,7 +30,7 @@ import whisk.common.{Config, Logging}
  * a value, and whose values are default values. A null value in the Map means there is
  * no default value specified, so it must appear in the properties file.
  * @param optionalProperties a set of optional properties (which may not be defined).
- * @param propertiesFile a File object, the whisk.properties file, which if given contains the property values.
+ * @param propertiesFile a File object, the org.apache.openwhisk.properties file, which if given contains the property values.
```
Comment (dgrove-oss) [inline common/scala/src/main/scala/org/apache/openwhisk/core/WhiskConfig.scala]: The name of the properties file should not be changed.

**Comment by houshengbo** [comment]
RangeError: Array buffer allocation failed","2018-10-25T20:28:08.370376749Z stderr: at Buffer.Uint8Array (native)
This error is occasionally raised by the test file ActionLimitsTests.

**Pull request merged**